docs/acls.md

```diff
@@ -1,4 +1,15 @@
-# ACLs use case example
+Headscale implements the same policy ACLs as Tailscale.com, adapted to the self-hosted environment.
+
+For instance, instead of referring to users when defining groups you must
+use users (which are the equivalent to user/logins in Tailscale.com).
```

Comment:
Is it intentional that this says "instead of referring to users ... you must use users" ? 😅